File: pset1/mario.c
#include<stdio.h>
#include<ctype.h>
int get_int(void);
int main(void)
{
    int i,j,k,h;
    do
    {
       printf("Height: ");    
       h = get_int();
    }while(h<0 || sizeof(h)!=4 || h>23);
    for(i=1;i<=h;i++)
    {
        for (j=0;j<h-i;j++)
        printf(" ");
        for(k=j;k<=h;k++)
        printf("#");
        printf("\n");
    }
    return 0;
}

File: pset6/pset6/mario.py
import cs50

while True:
    print("Height: ", end="")
    n = cs50.get_int()+1
    if n > 0 and  n < 23:
        break
i=1
s=n-1
for i  in range(i,n,+1):
    print(" "* (s-i),end="")
    print("#" * (i+1),end="")
    print()